org.eclipse.scanning.api/src/org/eclipse/scanning/api/device/DeviceResponse.java

```diff
@@ -80,14 +84,14 @@ public DeviceRequest process(DeviceRequest request) {
 				processRunnables(request, dservice);
 			}
 			return request;
-			
+
 		} catch (ModelValidationException ne) {
 			DeviceRequest error = new DeviceRequest();
 			error.merge(request);
 			error.setErrorMessage(ne.getMessage());
 			error.setErrorFieldNames(ne.getFieldNames());
 			return error;
-			
+
 		} catch (Exception ne) {
 			ne.printStackTrace();

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1148)